browser-extension/mv2/src/client/js/scriptRuleHandler.js

```diff
@@ -128,24 +128,98 @@ RQ.ScriptRuleHandler.includeJSScriptsInOrder = function (scripts, callback, inde
 };
 
 RQ.ScriptRuleHandler.includeJS = function (script, callback) {
+  if (!script.value) throw new Error("Script value is empty");
   if (script.type === RQ.SCRIPT_TYPES.URL) {
-    RQ.ClientUtils.addRemoteJS(script.value, callback);
+    if (script.attributes) {
+      RQ.ScriptRuleHandler.addRemoteJSWithAttributes(script.value, script.attributes, callback);
```

Comment:
We shouldn't create a separate function but instead just pass the attributes and the addRemoteJS function should check if there are attributes then add them otherwise not.

Reply:
updated this in a separate PR to make it easy for @nafees87n to also review it - https://github.com/requestly/requestly/pull/1400